requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 pip~=22.0
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@v0.37.0-rc0
```

Comment:
The requirements also need update.

Reply:
I'd agree with that, but in the [guidance docs](https://github.com/PennyLaneAI/guidance-docs/blob/master/development/lightning-release.md?plain=1#L75) we suggest it should be done after the release. This makes sense since `master` could have diverged and be incompatible at this point. So on the one hand, all we care about on the release branch (e.g. `v0.37.0_release`) is that wheels will work with PennyLane v0.37.0 (i.e. `v0.37.0-rc0`), but on the other hand we want this back to `master` when we merge the RC branch changes into `master`.

Reply:
We had problems with this last release.

Reply:
Was the resolution brought about changing to `master` or keeping to `v0.37.0-rc0`?